corehq/apps/userreports/views.py

```diff
@@ -509,6 +510,10 @@ def dispatch(self, request, *args, **kwargs):
             self.app_id = self.request.GET.get('application', None)
             self.source_id = self.request.GET['source']
             if self.registry_slug:
+                if not toggles.DATA_REGISTRY_UCR.enabled(self.domain):
```

Comment:
This check means domains without the flag won't be able to edit data registry UCRs either, right? (which I assume is what we want)